Add Product Specification Column Information (#10)
openmsupply/msupply_docs: content/health-supply-hub/submitting_bids.md

| B      | Internal Line ID – This is the unique ID number to link your quote/s back to the correct Quote Item when the Purchaser downloads quotes into their mSupply system. |
| C      | Item Code # – This is the code the Purchaser uses for this item                                     |
| D      | Item Name Column                                                                                    |

> | E      | Product Specification Column                                    |

Reviewer: ```suggestion
| E      | Product Specifications, as provided by the purchaser                                  |
```

> | F      | Number of Packs req uired in the Purchaser’s requested pack size                                    |

Reviewer: Looks like an extra space in `required` here?

Reviewer: ```suggestion
| F      | Number of Packs required in the Purchaser’s requested pack size                                    |
```

| G      | Purchaser's requested Pack Size                                                                     |
| H      | Total Quantity of single units being requested by the Purchaser                                    |
| I      | Any conditions the Purchaser may have for a particular Quote Item                                   |
| J      | Purchasers Unit of Measure                                                                          |

1. This screen shot shows the fields you must fill in to submit a quote

![Quote Template Supplier Section](/health-supply-hub/images/quote_supplier_fields.png)

| Column | Description                                                                                                   |
| ------ | ------------------------------------------------------------------------------------------------------------- | 
| K      | Enter in how many packs you are offering to supply the Purchaser                                              |
| L      | Enter in the Pack Size of the product your offering to supply                                                 |

> | M      | Enter in the Price you are offering to supply a single pack of your product at, in the currency you allocated in the next column                                           |

Reviewer: ```suggestion
| M      | Enter in the Price you are offering to supply a single pack of your product at, in the currency you specify in the next column                                           |
```
Some of this wording a is a bit convoluted, thought this might be slightly tidier...
